dev-docs/source/DataFilesForTesting.rst

```diff
@@ -131,6 +132,22 @@ This does the following:
 -  tells the user to upload the file(s),
    ``d6948514d78db7fe251efb6cce4a9b83``, to the `remote store <https://testdata.mantidproject.org/ftp/external-data/upload>`_
 
+2: Using the python script addTestData.py in the Testing/SystemTest/scripts directory. It can be called a follows:
+
+.. code-block:: sh
+
+   python addTestData.py --inputfiles file1.nxs file2.nxs file3.nxs --targetdirectory Testing/Data/UnitTest/
```

Comment:
Does the `--targetdirectory` need to be a full path? If so maybe we should put `<mantid_source>/Testing/Data/UnitTest/` or something similar instead